Dockerfile

```diff
@@ -16,9 +16,9 @@ MAINTAINER QuantConnect <contact@quantconnect.com>
 # RUN \
 #     cd /root/Lean && \
 #     sed -i 's/4.5/4.0/' Algorithm.VisualBasic/QuantConnect.Algorithm.VisualBasic.vbproj && \
-#     wget https://nuget.org/nuget.exe && \
+#     wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe && \
```

Comment:
The old link is working fine for me. Why was this change required/

Reply:
The error provided in the linked issue complains about nuget packages requiring version `2.1.2` and I've confirmed that the originally linked nuget installer is for `2.8.61717.93`.

Reply:
The errors in the #3711 states it requires `2.12` or above not `2.1.2` :)

Reply:
wow, further, nuget is _far_ beyond major version 2 and at about `5.3.x`